### PR TITLE
docs(Tabs): update README

### PR DESCRIPTION
**Pull request by sofiushko**

### CODE REVIEW

File: src/components/Tabs/README.md [hunk 1]
```diff
@@ -1,19 +1,178 @@
...
+    items={[
+        {id: 'first', title: 'extra large 1'},
+        {id: 'second', title: 'extra large 2'},
+    ]}
+/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Properties
```
Comment (korvin89) [inline src/components/Tabs/README.md:175]: Let's rename to "Tabs properties" to be able to distinguish from `Tabs.Item` properties

File: src/components/Tabs/README.md [hunk 2]
```diff
@@ -25,33 +184,131 @@
...
+```tsx
+<Tabs activeTab="first">
+  <Tabs.Item id="first" title="First Tab" onClick={() => {}} counter={13} />
+  <Tabs.Item id="second" title="Second Tab" onClick={() => {}} counter={3} />
+</Tabs>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Properties
```
Comment (korvin89) [inline src/components/Tabs/README.md:303]: I think it should have little bit bolder font like `Tabs` properties have

File: src/components/Tabs/README.md [hunk 3]
```diff
@@ -25,33 +184,131 @@
...
+
```
Comment (korvin89) [inline src/components/Tabs/README.md:304]: The same issue about order and composition of tabs

File: src/components/Tabs/README.md [hunk 4]
```diff
@@ -1,19 +1,178 @@
...
+        {id: 'second', title: 'extra large 2'},
+    ]}
+/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Properties
 
 | Name             | Type                                                                       | Required | Default      | Description                                                          |
```
Comment (korvin89) [inline src/components/Tabs/README.md:177]: Let's change the order and composition of tabs. U can take [Button](https://github.com/gravity-ui/uikit/blob/main/src/components/Button/README.md#properties) as example

File: src/components/Tabs/README.md [hunk 5]
```diff
@@ -1,19 +1,178 @@
...
+
+```tsx
+import {Tabs} from '@gravity-ui/uikit';
+```
+
+Tabs is used to explore, organize content and switch between different views.
+
+## Items
+
+For render tabs items you can use `items`.
```
Comment (amje) [inline src/components/Tabs/README.md:15]: > ... `items` property

File: src/components/Tabs/README.md [hunk 6]
```diff
@@ -1,19 +1,178 @@
...
+    activeTab="first"
+    items={[
+        {id: 'first', title: 'Third Tab'},
+        {id: 'second', title: 'Second Tab'},
+        {id: 'third', title: 'Third Tab', disabled: true},
+    ]}
+/>
+`}
+>
+    <UIKit.Tabs
```
Comment (amje) [inline src/components/Tabs/README.md:30]: This example won't be interactive. We need to use stateful example here. It is implemented in landing repo. See example of Popup docs

File: src/components/Tabs/README.md (hunk 1 above)
Comment (amje) [inline src/components/Tabs/README.md:175]: Properties section should always be the last section

File: src/components/Tabs/README.md [hunk 7]
```diff
@@ -25,33 +184,131 @@
...
...0 0 16 16"><path fill="currentColor" fill-rule="evenodd" d="M7.199 2H8.8a.2.2 0 0 1 .2.2c0 1.808 1.958 2.939 3.524 2.034a.199.199 0 0 1 .271.073l.802 1.388a.199.199 0 0 1-.073.272c-1.566.904-1.566 3.164 0 4.069a.199.199 0 0 1 .073.271l-.802 1.388a.199.199 0 0 1-.271.073C10.958 10.863 9 11.993 9 13.8a.2.2 0 0 1-.199.2H7.2a.199.199 0 0 1-.2-.2c0-1.808-1.958-2.938-3.524-2.034a.199.199 0 0 1-.272-.073l-.8-1.388a.199.199 0 0 1 .072-.271c1.566-.905 1.566-3.165 0-4.07a.199.199 0 0 1-.073-.271l.801-1.388a.199.199 0 0 1 .272-.073C5.042 5.138 7 4.007 7 2.2c0-.11.089-.199.199-.199ZM5.5 2.2c0-.94.76-1.7 1.699-1.7H8.8c.94 0 1.7.76 1.7 1.7a.85.85 0 0 0 1.274.735 1.699 1.699 0 0 1 2.32.622l.802 1.388c.469.813.19 1.851-.622 2.32a.85.85 0 0 0 0 1.472 1.7 1.7 0 0 1 .622 2.32l-.802 1.388a1.699 1.699 0 0 1-2.32.622.85.85 0 0 0-1.274.735c0 .939-.76 1.7-1.699 1.7H7.2a1.7 1.7 0 0 1-1.699-1.7.85.85 0 0 0-1.274-.735 1.698 1.698 0 0 1-2.32-.622l-.802-1.388a1.699 1.699 0 0 1 .622-2.32.85.85 0 0 0 0-1.471 1.699 1.699 0 0 1-.622-2.321l.801-1.388a1.699 1.699 0 0 1 2.32-.622A.85.85 0 0 0 5.5 2.2Zm4 5.8a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0ZM11 8a3 3 0 1 1-6 0 3 3 0 0 1 6 0Z" clip-rule="evenodd"></path></svg></svg>
```
Comment (amje) [inline src/components/Tabs/README.md:213]: There should be jsx, not raw SVG. Also extra svg wrapper can be omitted.

File: src/components/Tabs/README.md [hunk 8]
```diff
@@ -25,33 +184,131 @@
...
+  <Tabs.Item id="first" title="First Tab" onClick={() => {}} counter={13} />
+  <Tabs.Item id="second" title="Second Tab" onClick={() => {}} counter={3} />
+</Tabs>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Properties
+
+| Name     | Type                        | Required | Default | Description                      |
```
Comment (amje) [inline src/components/Tabs/README.md:305]: Order of columns is: Name, Description, Type, Default. Type and Default are centered

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/Tabs/README.md (hunk 2 above)
Comment (sofiushko) [reply src/components/Tabs/README.md:303]: Not sure about it
`Tabs.Item` is subtitle of `Tabs` and 
`Tabs.Item properties` is subtitle of `Tabs.Item` like in menu component https://github.com/gravity-ui/uikit/blob/main/src/components/Menu/README.md

File: src/components/Tabs/README.md (hunk 1 above)
Comment (sofiushko) [reply src/components/Tabs/README.md]: tabs has tabs.item component, I looked at menu component docs in this case

File: src/components/Tabs/README.md (hunk 6 above)
Comment (amje) [reply src/components/Tabs/README.md:30]: All examples where outer state needed

Comment (amje) [reply src/components/Tabs/README.md:30]: Oh, it seems every example has a state. Ок then, let's keep unchanged for now

File: src/components/Tabs/README.md (hunk 1 above)
Comment (amje) [reply src/components/Tabs/README.md]: Ok, Menu looks reasonable, let's follow the structure

File: src/components/Tabs/README.md (hunk 7 above)
Comment (amje) [reply src/components/Tabs/README.md:201]: Menu is not a reference for this)

File: src/components/Tabs/README.md (hunk 1 above)
Comment (sofiushko) [reply src/components/Tabs/README.md]: i moved Tabs properties to the end

File: src/components/Tabs/README.md (hunk 7 above)
Comment (sofiushko) [reply src/components/Tabs/README.md:201]: Ok, if we have reference for this? In button component is similarly

Comment (amje) [reply src/components/Tabs/README.md:201]: Button has correct SVG examples

File: src/components/Tabs/README.md [hunk 9]
```diff
@@ -1,57 +1,314 @@
...
+| activeTab        | Active tab ID                                                        |                                  `String`                                  |              |
+| allowNotSelected | Allows `activeTab` to be undefined                                   |                                 `Boolean`                                  |              |
+| items            | Tabs array                                                           |                             `TabsItemProps[]`                              |     `[]`     |
+| onSelectTab      | Select tab handler                                                   |                    `onSelectTab?(tabId: string): void`                     |              |
+| wrapTo           | Allows to wrap `TabItem` into another component or render custom tab | `wrapTo?(item: TabsItemProps, node: React.ReactNode, index: number): void` |              |
+| className        | CSS-class of element                                                 |                                  `String`                                  |              |
```
Comment (amje) [inline src/components/Tabs/README.md:314]: Let's use lowercase for all type primitives

File: src/components/Tabs/README.md (hunk 7 above)
Comment (sofiushko) [reply src/components/Tabs/README.md]: Paste from button)

File: src/components/Tabs/README.md (hunk 9 above)
Comment (sofiushko) [reply src/components/Tabs/README.md]: done

**Review by amje — APPROVED** [review]

**Pull request merged**